components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -3413,13 +3419,16 @@ private void doPatchGroup(String groupId, String currentGroupName, Map<String, L
             carbonUM.updateGroupName(currentGroupName, newGroupName);
 
         } catch (UserStoreException e) {
+            if (e instanceof org.wso2.carbon.user.core.UserStoreException && StringUtils
+                    .equals(UserCoreErrorConstants.ErrorMessages.ERROR_CODE_NON_EXISTING_USER.getCode(),
+                            ((org.wso2.carbon.user.core.UserStoreException) e).getErrorCode())) {
+                throw new BadRequestException(ResponseCodeConstants.INVALID_VALUE);
```

Comment:
why aren't we passing the throwable?

Reply:
```
public class BadRequestException extends AbstractCharonException {
    public BadRequestException() {
        this("invalid request");
    }

    public BadRequestException(String scimType) {
        this("Request is unparsable, syntactically incorrect, or violates schema.", scimType);
    }

    public BadRequestException(String details, String scimType) {
        super(400, details, scimType);
    }
}
```

This is the implementation of `BadRequestException`. There's no way to pass the throwable to the `BadRequestException`.

Reply:
Added a log with the throwable as suggested offline.